Validation/CaloTowers/test/macros/RelValHarvest_2023.py

```diff
@@ -48,55 +48,61 @@ def getDataSets( dsFlags = {'RelValMinBias_13__':'MinBias'},
                 if str in line:
                     # extract dataset path
                     path = line.split('\'')[1].strip()
-                    #print ("Getting DQM output from dataset: %s"%path)
-                    if (path.find("Ideal") > 0 or path.find("design") > 0 or path.find("FastSim") > 0 or path.find("DQM") < 0 or path.find("Pixel") > 0 ):  #skip for unnecessary samples
-                        continue
-                    print (path.split("/")[-1]) #path
-                    if printDS:
-                        continue
+                    #print("Getting DQM output from dataset: %s"%path)
+                 #   if (path.find("Ideal") > 0 or path.find("design") > 0  or path.find("DQM") < 0 or path.find("Pixel") > 0 or path.find("2021") > 0 or path.find("FastSim") > 0 or path.find("asymptotic") > 0):  #skip for unnecessary samples
+                    if (path.find("2023") < 0):
+                        continue    
+                        print(path.split("/")[-1]) #path
+                        if printDS:
+                           continue
 
                     # construct file name
                     fname = path.split("/")[-1]
-
+                    #print "smDebug2: ",fname
                     # create file name for use with hcal scripts
                     info = fname.split("__")[2].replace(label, "").strip("-")
-
+                    #print "smDebug3: ",info
                     #The Data sample have an additional piece put in. We strip it out so that the MC and DATA code can be common
 
-                    if camType == "DATA":
-                        iparts = info.split("_")
-                        info = ""
-                        skip = False
-                        for fragment in iparts:
-                            if skip:
-                                info = info.strip("_")
-                                skip = False
-                                continue
-                            if fragment == "RelVal":
-                                skip = True
-                                continue
-                            info += fragment
-                            info += "_"
-                        info = info.strip("_")
+                    # if camType == "DATA":
+                    #     iparts = info.split("_")
+		    #     info = ""
+		    #     skip = False
+		    #     for fragment in iparts:
+                    #         if skip:
+                    #             info = info.strip("_")
+                    #             skip = False
+                    #             continue
+                    #         if fragment == "RelVal":
+                    #             skip = True
+                    #             continue
+                    #         info += fragment
+                    #         info += "_"
+                    #     info = info.strip("_")
```

Comment:
May I ask why do you just comment out this part now?
I also see commented lines in a few other places. Could you please see if you can do a possible cleaning and leave some explanation for the part that is to be retained? Thanks!

Reply:
@tjavaid 
Actually it's not me who wrote (and commented these lines). 
I've got it from private files of the colleague, who has left CMS and HEP for industry. 
So, this part **needs to be revised in the future by a dedicated person (developer/maintainer)**,
so I'd like to keep them (as a kind of "document") to possibly reconsider it in the future.  
Right now it works as such for our actual purposes...   

If you really insist on removing some commented fragments in this /test area, I can do it.